packages/dart_frog_cli/lib/src/daemon/domain/dev_server_domain.dart

```diff
@@ -217,3 +223,9 @@ class DevServerDomain extends DomainBase {
     _devServerRunners.clear();
   }
 }
+
+bool _isValidHost(dynamic host) {
+  return host is String &&
+      RegExp(r'^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$')
```

Comment:
will "localhost" pass this test?

Reply:
No.

Reply:
Yes, in [this line](https://github.com/VeryGoodOpenSource/dart_frog/pull/924/files#diff-aa1e9275509af92caf516f301386d49ca1ba051870bdee3c26091c44e07730b3R46) we are passing `localhost` to this function in case of nullability.